FinniversKit/Sources/Components/FrontPageTransactionView/FrontPageTransactionView.swift

```diff
@@ -0,0 +1,201 @@
+import UIKit
+
+public protocol FrontPageTransactionViewDelegate: AnyObject {
+    func transactionViewTapped(_ transactionView: FrontPageTransactionView)
+}
+
+public class FrontPageTransactionView: UIView {
+    private let imageWidth: CGFloat = 56
+    private let cornerRadius: CGFloat = 8
+    
+    private lazy var titleLabel: UILabel = {
+        let label = Label(withAutoLayout: true)
+        label.font = .bodyStrong
+        label.textColor = .licorice
+        label.numberOfLines = 1
+        label.textAlignment = .left
+        return label
+    }()
+    
+    private lazy var subtitleLabel: UILabel = {
+        let label = Label(withAutoLayout: true)
+        label.font = .body
+        label.textColor = .licorice
+        label.numberOfLines = 1
+        label.textAlignment = .left
+        label.widthAnchor.constraint(equalToConstant: 228).isActive = true
```

Comment:
Can the constraints be moved into `setup()` ?

Reply:
done 😄